examples/bare-metal/ic/README.rst

```diff
@@ -2,10 +2,9 @@
 ic
 ===============
```

Comment:
In other places, the length of the header formatting marks matches the length of the header text.  It looks like it does not do so in some of the bare-metal readme files.  Regardless, the documentation produced looks just fine, so any change is very much low priority.